internal/k6runner/http.go

```diff
@@ -106,14 +109,18 @@ func (r HttpRunner) Run(ctx context.Context, script Script) (*RunResponse, error
 		waitRemaining := max(0, wait-time.Since(start))
 		r.logger.Warn().Err(err).Dur("after", waitRemaining).Msg("retrying retryable error")
 
+		// Discard context, backoffSpan needs no children.
```

Comment:
Sorry, you lost me. If it has no children, and it's not adding any data, why is it interesting?

Reply:
I think it might answer the question of _what_ is happening between requests. Not a hard opinion there, if you think it's obvious I can drop it.

Reply:
To be fair, I haven't added backoff spans on other places so I might as well drop this one and avoid the confusion entirely. We can add it later if it becomes a problem.